core/services/job/mocks/spec.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v2.2.1. DO NOT EDIT.
+// Code generated by mockery v2.1.0. DO NOT EDIT.
```

Comment:
@spooktheducks this number probably shouldn't go backwards. Try updating your local mockery version?